clicommand/pipeline_upload.go

```diff
@@ -227,35 +233,46 @@ var PipelineUploadCommand = cli.Command{
 			l.Fatal("Pipeline parsing of \"%s\" failed (%s)", src, err)
 		}
 
-		// In dry-run mode we just output the generated pipeline to stdout
-		if cfg.DryRun {
-			enc := json.NewEncoder(os.Stdout)
-			enc.SetIndent("", "  ")
-
-			// Dump json indented to stdout. All logging happens to stderr
-			// this can be used with other tools to get interpolated json
-			if err := enc.Encode(result); err != nil {
-				l.Fatal("%#v", err)
-			}
-
-			return
-		}
-
 		if len(cfg.RedactedVars) > 0 {
 			needles := redaction.GetKeyValuesToRedact(shell.StderrLogger, cfg.RedactedVars, env.FromSlice(os.Environ()).ToMap())
 			serialisedPipeline, err := result.MarshalJSON()
 
 			if err != nil {
-				l.Fatal("Couldn’t scan the %q pipeline for redacted variables. This parsed pipeline could not be serialized, ensure the pipeline YAML is valid, or ignore interpolated secrets for this upload by passing --redacted-vars=''. (%s)", src, err)
+				l.Fatal("Couldn't scan the %q pipeline for redacted variables. This parsed pipeline could not be serialized, ensure the pipeline YAML is valid, or ignore interpolated secrets for this upload by passing --redacted-vars=''. (%s)", src, err)
```

Comment:
Not a fan of the pretty apostrophes?

Reply:
eh.... when i initially did this, i told myself it was for terminal compatibilty (and because my editor pointed it out to me), but realistically i think we can trust that basically everyone is using a terminal that is UTF-8 compatible. i also scatter emoji all throughout my codebases too, so i'm a bit of a hypocrite here :P

i'll revert it back to using the fancy apostrophe